packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/Bootstrap.tsx

```diff
@@ -1,21 +1,24 @@
+import { useEffect } from 'react';
 import { SitecorePageProps } from 'lib/page-props';
-import { context } from 'src/lib/context';
+import { CloudSDK } from '@sitecore-cloudsdk/core/browser';
+import '@sitecore-cloudsdk/events/browser';
 import config from 'temp/config';
 
 /**
  * The Bootstrap component is the entry point for performing any initialization logic
  * that needs to happen early in the application's lifecycle.
  */
 const Bootstrap = (props: SitecorePageProps): JSX.Element | null => {
-  /**
-   * Initializes the application Context and associated Software Development Kits (SDKs).
-   * This function is the entry point for setting up the application's context and any SDKs that are required for its proper functioning.
-   * It prepares the resources needed to interact with various services and features within the application.
-   */
-  context.init({
-    siteName: props.site?.name || config.sitecoreSiteName,
-    pageState: props.layoutData?.sitecore?.context?.pageState,
-  });
+  // Browser ClientSDK init allows for page view events to be tracked
+  useEffect(() => {
```

Comment:
Few things:
1) I noticed one thing while I was testing angular cloudsdk initialization. I think we need to skip it if we are in preview/edit mode. If it's normal mode we can initialize.
2) We don't have any conditions here regarding 'window', 'dev/prod mode': see [here](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/lib/context/sdk/events.ts#L9-L12). As far as I remember "useEffect" is not executed on server side, this case should be fine, while for dev vs prod we still need to have a check

Don't forget to update Migration guide